src/components/loader/Loader.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC } from "react";
+import "./Loader.css";
+
+const Loader: FC = () => {
+  return (
+    <div className="loader-wrapper">
+      <div className="loader"></div>
```

Comment:
```suggestion
      <div className="loader" />
```
мелочь, но просто привыкай постепенно, что дивы и прочее можно тоже одним тегом писать если они пустые